context/app/static/js/components/Detail/Visualization/style.js

```diff
@@ -36,14 +28,16 @@ const StyledSectionContainer = styled(SectionContainer)`
 
 const ExpandButton = styled(Button)`
   display: inline-block;
+  background-color: #ffffff;
   margin-left: 0.5rem;
   svg {
     vertical-align: middle;
   }
 `;
 
 const SelectionButton = styled(Button)`
-  color: black;
+  margin-left: 0.5rem;
```

Comment:
```suggestion
  margin-left: ${(props) => props.theme.spacing(1)}px;
```

Reply:
Material has us using `px` for spacing. There are arguments both ways, but I think we should be consistent.